scripts/remote_build_entrypoint.sh

```diff
@@ -105,6 +105,37 @@ if [ "$rc" -ne "0" ]; then
   exit 0
 fi
 
+# NOTE: do this here so the buildenv-sidecar doesn't need to unpack the squashfs file to do it later
+echo "Copying SMS CA Public Certificate to target image root"
+mkdir -p "${IMAGE_ROOT_DIR}etc/cray"
+cp -r /etc/cray/ca "${IMAGE_ROOT_DIR}etc/cray/"
+RC=$?
+if [[ ! $RC ]]; then
```

Comment:
I don't think this will do what you want it to. I am pretty sure this will be false if RC has any numeric value. I think you want:
```suggestion
if [[ $RC -ne 0 ]]; then
```

I see this same issue exists earlier in the file. It isn't related to your PR, but we probably should review all of the IMS shell scripts for this error. Doesn't have to be with this PR, of course.